bin/cli.js

```diff
@@ -154,7 +154,7 @@ async.waterfall([
 
     if (_.includes(documentedVersions, version)) {
       console.log(`Omitting: ${version}`);
```

Comment:
Now that we have the debug package should we start moving `console.logs` to it? Or best to do it all at once?